script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -661,7 +670,7 @@ If I don't put on my glasses, I'm going to lose my mind.
 [sha:f1cef6fe3a289d3bfc0af9176bdeb36533660e09]{
 -- Page 316, Offset 17359.
 -- 喉元までせり上がった嘔吐感をこらえながら、震える手で眼鏡をかけ直す。
-Fighting back the urge to vomit that has welled up in my throat,
+Fighting back the urge to vomit that is welling up in my throat,
```

Comment:
Welled is past in JP, but we've definitely muddled tenses in the past. If we presentify it can probably drop the `that is`

Reply:
this should be present given the previous and next line imo